src/common/points.h

```diff
@@ -1056,27 +1056,9 @@ void init_by_array(sfmt_state_t *s, uint32_t *init_key, int key_length)
 {
   int i, j, count;
   uint32_t r;
-  int lag;
-  int mid;
+  int lag = 11;
   int size = N * 4;
```

Comment:
I don't think this is correct. N is defied as:

#define N (MEXP / 128 + 1)

And MEXP comes from an include definition which may depends on the platform.

size = N * 4 is 155 if MEXP is set to default (see top of this file), and so path "size  >= 60" will be taken.